src/app/portlets/content-types/fields/content-type-fields-add-row/content-type-fields-add-row.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="content-type-fields_add-row__container">
+    <div class="content-type-fields-add-row" 
+        [ngClass]="{'content-type-fields-add-row__expanded': addRowSelected === 'expanded', 'content-type-fields-add-row__collapsed': addRowSelected === 'collapsed'}">    
+        <button type="button" pButton flat 
+            label="{{i18nMessages['contenttypes.content.add_rows']}}" (click)="addRow()" 
```

Comment:
I think a better key is contenttypes.dropzone.rows.add

Reply:
agree